### PR TITLE
Forward-port 8.2.2 changelog to main

### PR DESCRIPTION
**Pull request by andresrc**
See #31751

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -41,17 +41,18 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
...
 
 - Do not emit error log when filestream reader reaches EOF and `close.reader.on_eof` is enabled. {pull}31109[31109]
 - sophos.xg: Update module to handle new log fields. {issue}31038[31038] {pull}31388[31388]
 - Fix MISP documentation for `var.filters` config option. {pull}31434[31434]
-- Remove invalid "network_traffic" term from event.category. {pull}31674[31674]
 - Fix type mapping of client.as.number in okta module. {pull}31676[31676]
 
+*Filebeat*
+
+
```
Comment (bmorelli25) [inline CHANGELOG.next.asciidoc:55]: This is a duplicate of the above **Filebeat** section

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -121,10 +122,6 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 *Metricbeat*
 
-- Extend documentation about `orchestrator.cluster` fields {pull}30518[30518]
-- Enhance Oracle Module: Change tablespace metricset collection period {issue}30948[30948] {pull}31259[#31259]
-- Add orchestrator cluster ECS fields in kubernetes events {pull}31341[31341]
-- Add new Kubernetes module dashboards {pull}31341[31341]
```
Comment (bmorelli25) [inline CHANGELOG.next.asciidoc:127]: It looks like these three are all for 8.3. Do we want to delete them from this _next_ changelog?

Comment (andresrc) [reply CHANGELOG.next.asciidoc:127]: The thing is that they are in the 8.2.1 changelog, probably a bad backport, I'll take a look

Comment (andresrc) [reply CHANGELOG.next.asciidoc]: Fixed in a later commit for `CHANGELOG.next`, the changelog will be fixed in a follow-up PR

Comment (bmorelli25) [reply CHANGELOG.next.asciidoc]: Sounds good!

**Review by bmorelli25 — APPROVED** [review]

**Comment by andresrc** [comment]
@Mergifyio backport 8.3

**Pull request merged**